site2/docs/deploy-bare-metal.md

```diff
@@ -126,8 +126,12 @@ Directory | Contains
 
 ## Install Built-in Connectors (optional)
 
-> Since Pulsar release `2.1.0-incubating`, Pulsar provides a separate binary distribution, containing all the `built-in` connectors.
-> To enable the `built-in` connectors (optional), you can follow the instructions below.
+:::note
+
+Since Pulsar release `2.1.0-incubating`, Pulsar provides a separate binary distribution, containing all the `built-in` connectors.
+To enable the `built-in` connectors (optional), you can follow the instructions below.
+
+:::
```

Comment:
I'm considering we can improve the expression here since it's long time after `2.1.0-incubating` released.

Reply:
Make sense, I'll remove them.